geometric/internal.py

```diff
@@ -1684,7 +1684,7 @@ def convert_angstroms_degrees(prims, values):
 
 CacheWarning = False
 
-class InternalCoordinates(object):
+class InternalCoordinates(PrimitiveCoordinate):
```

Comment:
You are right. I don't exactly remember why I made this change. I changed it back to `object` from `PrimitiveCoordinate`. This does not affect the NEB since it does not inherit anything.